Reply-To feature

mwenell opened the pull request:
Insert Reply-To header into emails. This is useful when using "send only" mail server.

Use the plugin as follows to avoid spam filters:
- Use SPF for the domain
- Use DKIM in email server (optional)
- Use server's domain name as email address of sender, e.g. your_service_name@example.com
- Use Reply-To email address which you can read and reply, e.g. your_service_name@gmail.com

mwenell committed:
Reply-To feature

Tmeister commented:
Hi @mwenell and thank you for the pull request.

I check your code and I found some bugs.

1) You add duplicate code on the `class-wp-simple-mail-sender.php` and `admin/class-wp-simple-mail-sender-admin.php` there is no need to add the same hook and function in both files.

https://github.com/Tmeister/wp-simple-email-sender/pull/1/files#diff-43d5315418ee01ccfba0a34ea88c45aeR318

https://github.com/Tmeister/wp-simple-email-sender/pull/1/files#diff-0e77afe5d23c9e772c0212a1295e3e5aR156

2) In your function change_reply the is an undefined variable called $replyRegex and this generates a Notice and a Warning message.

https://github.com/Tmeister/wp-simple-email-sender/pull/1/files#diff-43d5315418ee01ccfba0a34ea88c45aeR325

Again, I appreciate your pull request but it needs some fixes, please, if you want to fix those errors create a new pull request.

Enrique.

mwenell commented:
HI,

Thanks for checking my fixes. I needed this feature and did not find this feature in your and this other similar plugin.

Your feedback is good, because i did not have time to understand deeply how your code worked as well as testing my own code, i see ;-)

I'll fix this.

However, i would like to see these as part of your future code and install this plugin from Wordpres.org, if possible.

mwenell commented:
I have made new version where i have fixed the bugs you reported. However, i founded some other bugs and fixed them same time as well as inserted Reply-To name feature.

How does it look now?
